docs/content/concepts/rest/rest-api.md

```diff
@@ -23,6 +23,11 @@ under the License.
 -->
 
 <body>
-<redoc spec-url="https://raw.githubusercontent.com/apache/paimon/master/paimon-open-api/rest-catalog-open-api.yaml"></redoc>
-<script src="https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js"></script>
+    <div id="redoc-container"></div>
+    <script src="/docs/master/js/redoc.standalone.js"></script>
```

Comment:
Refer it just like `"{{.Site.BaseURL}}/js/flink.js"`?

Reply:
After setting <script src="{{.Site.BaseURL}}/js/redoc.standalone.js"></script>, throw following error:
```
GET http://localhost:1313/docs/master/concepts/rest/rest-api/%7B%7B.Site.BaseURL%7D%7D/js/redoc.standalone.js net::ERR_ABORTED 404 (Not Found)
```

Reply:
Maybe add this `<script` to head.html, and this file just refer js function?